ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Runner.scala

```diff
@@ -79,7 +80,10 @@ final class Runner[T <: ReadWriteService, Extra](
     )
     implicit val materializer: Materializer = Materializer(actorSystem)
 
-    val sharedEngine = Engine.StableEngine()
+    val sharedEngine =
```

Comment:
rather than hardcoding a different value here, should we just pass it through and make it configurable?

Reply:
Yes please. I would prefer if this was opt-in with a CLI argument.

Reply:
should the option be hidden ?

Reply:
Eh, this runner is still intended mostly for people who know what they're doing, so it's probably fine.

(It was never intended to actually be used, but c'est la vie.)